plugin/cmd/kube-scheduler/app/server.go

```diff
@@ -80,13 +81,22 @@ func (s *SchedulerServer) Run(_ []string) error {
 	}
 
 	go func() {
+		mux := http.NewServeMux()
 		if s.EnableProfiling {
```

Comment:
FWIW, there shouldn't be an overhead to having this unless it is used (as in, queried). One downside is that exposing it publicly could allow an avenue for DOSing through it. I don't think that's a big concern today though.